src/popup/pages/TermsOfService.vue

```diff
@@ -949,10 +950,20 @@ import {
 
 export default {
   components: { ChevronDownIcon },
+  setup() {
+    const translations = [];
+
+    for (let i = 1; i < 12; i += 1) {
+      translations.push(i18n.global.t(`pages.termsOfService.section${i}Title`));
+    }
+
+    return {
+      translations,
+    };
+  },
   data() {
     return {
-      details: Object.entries(this.$t('pages.termsOfService'))
-        .filter(([k]) => /section[0-9]+Title/.test(k))
+      details: Object.entries(this.translations ?? [])
```

Comment:
I suggest to avoid mixing the same logic between composition API and options API. Maybe using a computed property would solve this issue?

Reply:
Will create a JIRA task about this refactor.